### PR TITLE
cat1315 disable undo button disabled after switching when editing text

### PR DESCRIPTION
**Pull request by StefanBuerscher**
ok to test

### CODE REVIEW

**Comment by avihric** [comment]
Stick to commit guideline please. :eyes:  [wiki](https://github.com/Catrobat/Catroid/wiki/Setup-working-environment#commit-guidelines)

File: catroid/src/org/catrobat/catroid/ui/fragment/FormulaEditorFragment.java [hunk 1]
```diff
@@ -97,6 +97,7 @@
 	private FormulaBrick clonedFormulaBrick;
 	private Brick.BrickField currentBrickField;
 	private Formula currentFormula;
+	private Menu currentMenu;
```
Comment (obusher) [inline catroid/src/org/catrobat/catroid/ui/fragment/FormulaEditorFragment.java]: I think we do not need this member.

Comment (StefanBuerscher) [inline catroid/src/org/catrobat/catroid/ui/fragment/FormulaEditorFragment.java]: Ok, then I dont know how to solve this.

Comment (obusher) [inline catroid/src/org/catrobat/catroid/ui/fragment/FormulaEditorFragment.java]: I did not expand the `github-diff (@@ -483,6 +486,14 @@)`, so it looked like the `currentMenu` variable gets used in `onPrepareOptionsMenu`.

Comment (avihric) [inline catroid/src/org/catrobat/catroid/ui/fragment/FormulaEditorFragment.java]: @obusher :trollface:  :grin:

File: catroid/src/org/catrobat/catroid/ui/fragment/FormulaEditorFragment.java [hunk 2]
```diff
@@ -483,6 +486,14 @@ public boolean saveFormulaIfPossible() {
...
 		FormulaElement formulaParseTree = formulaToParse.parseFormula();
 
+		MenuItem undo = currentMenu.findItem(R.id.menu_undo);
+		undo.setIcon(R.drawable.icon_undo_disabled);
+		undo.setEnabled(false);
+
+		MenuItem redo = currentMenu.findItem(R.id.menu_redo);
+		redo.setIcon(R.drawable.icon_redo_disabled);
+		redo.setEnabled(false);
+
```
Comment (obusher) [inline catroid/src/org/catrobat/catroid/ui/fragment/FormulaEditorFragment.java]: I would put this code on line [473](https://github.com/Catrobat/Catroid/pull/1294/files#diff-d5336e4f047445d9085f6c015edeb69eR473) in method `setInputFormula()`. It has nothing to do with `saveFormulaIfPossible()`.

**Comment by obusher** [comment]
@StefanBuerscher Did i scare you away ? :smile_cat: 
Could please do those little changes.. TY

**Comment by StefanBuerscher** [comment]
I will do it in the next days, I was not home for the last few days.

Am 2015-07-29 um 10:16 schrieb F®@nz S©h®eỉne®:

**Comment by StefanBuerscher** [comment]
#1316

**Pull request closed without merging**